gallery.js

```diff
@@ -0,0 +1,21 @@
+let largeImg = document.getElementById('largeImg');
+let thumbs = document.getElementById('thumbs');
+thumbs.onclick = function(event) {
+  console.log('click');
+  let target = event.target;
+  while (target != this) {
```

Comment:
There are the more elegant solution then while
https://developer.mozilla.org/en-US/docs/Web/API/Element/closest